usaspending_api/awards/v2/views/funding_rollup.py

```diff
@@ -16,10 +16,7 @@
     with gather_financial_accounts_by_awards as (
         WITH closed_submissions AS (
```

Comment:
Is `closed_submission` necessary? Look at the SQL changes in #2793

Reply:
It is not. The SQL has been updated, and there might be some other queries throughout the codebase we can update in the tech debt sprint.